eidas_node/tests/constants.py

```diff
@@ -6,6 +6,7 @@
 KEY_FILE = str(DATA_DIR / 'key.pem')
 CERT_FILE = str(DATA_DIR / 'cert.pem')
 NIA_CERT_FILE = str(DATA_DIR / 'nia-test-cert.pem')
+NIA_CERT_FILE_LATEST = str(DATA_DIR / 'nia-test-cert-latest.pem')
```

Comment:
This doesn't seem to be used anywhere. Am I missing something?

Reply:
It will be used in python-env.

Reply:
Then it should be there and not here :/

Reply:
Only the constant or also the file?

Reply:
Both

Reply:
Removed.